gallery/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@sonatype/react-shared-components-gallery",
-  "version": "9.4.0",
+  "version": "9.4.1",
```

Comment:
This is a new feature so a minor version

Reply:
> This is a new feature so a minor version

Ooops: https://github.com/sonatype/sonatype-react-shared-components/pull/586/commits/fa06b6ae31bb74edcab63d2234a8b9dd70536295